### ISSUE TITLE
OSGi manifest for the 1.6.0 release uses SNAPSHOT in the version

### ISSUE THREAD

**Issue by davsclaus** [everit-org/json-schema#110]
I am using the 1.6.0 release and inside this JAR we see SNAPSHOT
```
Manifest-Version: 1.0
Bnd-LastModified: 1500586724298
Build-Jdk: 1.8.0_131
Built-By: jitpack
Bundle-Description: Implementation of the JSON Schema Core Draft v4 sp
 ecification built with the org.json API
Bundle-DocURL: http://www.everit.org
Bundle-License: http://www.apache.org/licenses/LICENSE-2.0.txt
Bundle-ManifestVersion: 2
Bundle-Name: everit-org/json-schema
Bundle-SymbolicName: org.everit.json.schema
Bundle-Vendor: Everit Kft.
Bundle-Version: 1.6.0.SNAPSHOT
Created-By: Apache Maven Bundle Plugin
Export-Package: org.everit.json.schema;version="1.6.0.SNAPSHOT";uses:=
 "org.json",org.everit.json.schema.loader;version="1.6.0.SNAPSHOT";use
 s:="org.everit.json.schema,org.json"
Import-Package: com.damnhandy.uri.template;version="[2.1,3)",com.googl
 e.common.collect;version="[22.0,23)",com.google.common.net;version="[
 22.0,23)",org.apache.commons.collections;version="[3.2,4)",org.apache
 .commons.validator.routines;version="[1.6,2)",org.json;version="[2017
 0516.0,20170517)"
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
Tool: Bnd-3.2.0.201605172007

```

That should of course have been replaced to the 1.6.0 release number in the release build.

**Comment by erosb** [comment]
Thanks @davsclaus . Is it a blocking problem for you or is it fine if I release the next version without `.SNAPSHOT` when the time comes?

**Comment by davsclaus** [comment]
No its not blocking. 

The bundle can still be loaded even with 1.6.0.SNAPSHOT.

**Comment by erosb** [comment]
I've just released version 1.6.1 . The export-package header doesn't contain the SNAPSHOT suffix this time. Closing this ticket.
